src/Schema/Comparator.php

```diff
@@ -309,17 +307,17 @@ public function diffTable(Table $fromTable, Table $toTable): ?TableDiff
 
         return new TableDiff(
             $fromTable,
-            $addedColumns,
+            array_values($addedColumns),
```

Comment:
> The return types of all methods except for getRenamed*() have been relaxed from list<T> to array<T>. The former implies that the elements of the list can be accessed by index and their order matters while this is not intended (those are just sets).

Why is this call to `array_values` needed now that this type has been relaxed?

Reply:
I still didn't want the keys of the associative arrays to leak into the diff, and some tests relied on the numeric keys for assertions. But this can be fixed.